ui/src/components/RORForm/index.tsx

```diff
@@ -232,6 +233,11 @@ const RORForm: React.FC<FormProps> = (props): React.ReactElement => {
             props.type == 'funders'
                 ? updateFunders([...funders, createdRorRecord])
                 : updateAffiliations([...affiliations, createdRorRecord]);
+
+            if (props.type === 'affiliations') {
+                updateConfirmNoAffliations(false);
```

Comment:
Is there a reason why updating this boolean when adding an affiliation and not when the user clicks the 'Yes' radio button? 

User can click 'Yes' and save the publication without adding any affiliations and this flag will remain true. Then if the user clicks 'No' again, the confirmation checkbox is selected by default, because this flag remained true all this time.